index.js

```diff
@@ -130,15 +133,24 @@ export default function ({
 				swaConfig.routes.push(
 					{
 						route: '/index.html',
-						rewrite: ssrFunctionRoute
+						rewrite: ssrFunctionRoute,
+						allowedRoles: allowedRoles
 					},
 					{
 						route: '/',
-						rewrite: ssrFunctionRoute
+						rewrite: ssrFunctionRoute,
+						allowedRoles: allowedRoles
 					}
 				);
 			}
 
+			swaConfig.routes.push(
+				{
+					route: '*',
+					allowedRoles: allowedRoles
+				}
+			);
+
```

Comment:
Instead of remembering to set `allowedRoles` on each route, can we apply them all at once? e.g.

```js
swaConfig.routes = swaConfig.routes.map((r) => ({ ...r, allowedRoles }));
```